src/main/java/in/twizmwaz/cardinal/module/modules/wools/WoolObjective.java

```diff
@@ -145,11 +145,7 @@ public void onWoolPickup(InventoryClickEvent event) {
                         this.touched = true;
                         if (touchMessage) {
                             double newProx;
-                            if (location != null) {
-                                newProx = location.distance(place.getVector());
-                            } else {
-                                newProx = player.getLocation().toVector().distance(place.getVector());
-                            }
+                            newProx = player.getLocation().toVector().distance(place.getVector());
```

Comment:
@Alfalik No, the functions of picking up a wool off the ground and taking it out of a chest should be the same. In the case of maps like Dynamo, wool comes from a spawner and you have to pick it up.